sdk/core/azure-core/src/main/java/com/azure/core/util/serializer/JacksonAdapter.java

```diff
@@ -23,7 +27,14 @@
  */
 public class JacksonAdapter implements SerializerAdapter {
     private static final Pattern PATTERN = Pattern.compile("^\"*|\"*$");
-    private final ClientLogger logger = new ClientLogger(JacksonAdapter.class);
+    private static final ClientLogger LOGGER = new ClientLogger(JacksonAdapter.class);
+
+    private static boolean useAccessHelper;
+
+    static {
+        useAccessHelper = Boolean.parseBoolean(Configuration.getGlobalConfiguration()
+            .get("AZURE_JACKSON_ADAPTER_USE_ACCESS_HELPER"));
```

Comment:
perhaps slightly out of the scope of this PR: should we add a doc somewhere? maybe a wiki page?

Reply:
Yes, I'll get that added somewhere. It'll likely be a larger update for the undocumented configurations that exist.